CHANGELOG.md

```diff
@@ -26,6 +26,11 @@ Changes are grouped as follows
 ### Fixed
 - Calls to datapoints `retrieve_dataframe` with `complete="fill"` would break using Pandas version 1.1.0 because it raises TypeError when calling `.interpolate(...)` on a dataframe with no columns.
 
+## [2.2.0] - 2020-08-13
```

Comment:
You also need to bump https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/__init__.py